pkg/apis/observability/v1alpha1/types.go

```diff
@@ -94,5 +120,5 @@ type OperatorsSpec struct {
 	// When the capability is enabled, the install is set to true, otherwise it is set to false.
 	// +optional
 	// +kubebuilder:validation:Optional
-	Install bool `json:"install,omitempty"`
+	Install *bool `json:"install,omitempty"`
```

Comment:
I think this is related to Jan's comment  and I am not sure I understand it (and yes I probably missed that in the previous PR so apologies). I thought when a capability is enabled then everything required is installed. It's not the case?

Reply:
Correct, when a capability is enabled, the operator installs the required operators.